util/db/repository.go

```diff
@@ -35,6 +35,8 @@ const (
 	tlsClientCertData = "tlsClientCertData"
 	// The name of the key storing the TLS client cert key in the secret
 	tlsClientCertKey = "tlsClientCertKey"
+	// The name of the key storing the GitHub App private key in the secret
+	githubAppPrivateKey = "githubAppPrivateKey"
```

Comment:
You already know this - but a reminder, rename `githubAppPrivateKey` to `githubAppPrivateKeySecret` to more clearly reflect that this is a reference to a `Secret`.

Reply:
@diranged Note that your suggested change was not made:

https://github.com/argoproj/argo-cd/blob/master/util/db/repository.go#L40

@stlava Other possible occurrences that have not been converted:

https://github.com/argoproj/argo-cd/search?q=githubAppPrivateKey